setup.py

```diff
@@ -6,12 +6,7 @@
       author='Dave Hunt',
       author_email='dhunt@mozilla.com',
       url='https://github.com/mozilla/pytest-mozwebqa',
-      packages=['pytest_mozwebqa'],
-      py_modules=[
```

Comment:
Good question. See [this documentation](https://docs.python.org/2/distutils/setupscript.html#listing-individual-modules). It's just less maintenance this way.